source/install/cloud-changelog.md

```diff
@@ -4,12 +4,116 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
+ - The [template picker](https://docs.mattermost.com/boards/work-with-boards.html#choose-a-board-template) has been improved to make it easier for users to find the best template for their project.
+
+#### ServiceNow Integration
+ - ServiceNow customers can now access and share their ServiceNow data from within Mattermost.
+ 
+#### ServiceNow Virtual Agent Integration
+ - The [Mattermost ServiceNow Virtual Agent Integration](https://www.loom.com/share/36f60299c6c349729160d6d8bce75c3c) makes it easy for employees and customers to resolve issues fast.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated prepackaged version of plugins affected by React 17 upgrade.
+ - Corrected in-product System Console legacy link to the Cloud Administrator's Guide.
+ - Updated prepackaged NPS version to 1.3.1.
+ - Updated prepackaged version of Apps plugin to 1.2.0.
+ - Added group members count to the group autocomplete.
+ - Clicking a group mention now displays group details and membership.
+ - Improved the collapsed state of the post formatting toolbar.
+ - App Framework - channel and user fields now support multiselect property, to allow users to select multiple values in the form.
+ - Increased the notification length on Windows to 120 from 50.
+ - Prioritized members of recently viewed DMs/GMs when adding users to a channel.
+ - Updated in-product confirmation modal for ``@here`` mentions to clarify that people & timezone counts don't include the current user.
+ - Added support for multiple users and channels to the ``/invite`` slash command.
+ - Downgraded French language support to Beta.
+
+#### Administration
+ - The export file now contains the server version and a creation timestamp.
+ - Plugins with a webapp component may need to be updated to work with Mattermost and the updated ``React v17`` dependency. This is to avoid plugins crashing with an error about ``findDOMNode`` being called on an unmounted component. While our `starter template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ depended on an external version of ``React``, it did not do the same for ``ReactDOM``. Plugins need to update their ``webpack.config.js`` directives to externalize ``ReactDOM``. For reference, see https://github.com/mattermost/mattermost-plugin-playbooks/pull/1489. Server-side only plugins are unaffected. This change can be done for existing plugins any time prior to upgrading to Mattermost and is backwards compatible with older versions of Mattermost.
+ - Added ``acknowledgements`` field to the post's metadata.
+ - Added the ability for customers to view their upcoming invoice at the **System Console > Billing & Account > Subscriptions** page.
+ - **Total Activated Users** was changed back to **Total Active Users** in **System Console > Reporting > Site Statistics**.
+ - The import job now logs the progress of the import.
+ - Added ``restore_group`` permission to the mmctl and to the **System Console > Permissions**.
```

Comment:
@amyblais - Can you point me to the PR for this update, please? I don't yet have this on my radar for docs updates. Thanks!

Reply:
PRs: https://github.com/mattermost/mattermost-server/pull/21806 & https://github.com/mattermost/mattermost-webapp/pull/11760.